2018-08-13 22:08:53,085 ERROR: Exception on /index [GET] [in c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Raiden\Documents\Python Projects\freespeak\app\routes.py", line 41, in index
    return render_template('index.html', title="Home", form=form, posts=posts.items, next_url=next_url, prev_url=prev_url)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Raiden\Documents\Python Projects\freespeak\app\templates\_post.html", line 12, in template
    said {{ moment.(post.timestamp).fromNow() }}:
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "C:\Users\Raiden\Documents\Python Projects\freespeak\app\templates\index.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "C:\Users\Raiden\Documents\Python Projects\freespeak\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block "doc"
    {%- block html %}
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block "html"
    {% block body -%}
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block "body"
    {% block content -%}
  File "C:\Users\Raiden\Documents\Python Projects\freespeak\app\templates\base.html", line 53, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\Raiden\Documents\Python Projects\freespeak\app\templates\index.html", line 11, in block "app_content"
    {% include '_post.html' %}
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Raiden\Documents\Python Projects\freespeak\app\templates\_post.html", line 12, in template
    said {{ moment.(post.timestamp).fromNow() }}:
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 875, in subparse
    add_data(self.parse_tuple(with_condexpr=True))
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 620, in parse_tuple
    args.append(parse())
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 432, in parse_expression
    return self.parse_condexpr()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 437, in parse_condexpr
    expr1 = self.parse_or()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 450, in parse_or
    left = self.parse_and()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 459, in parse_and
    left = self.parse_not()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 470, in parse_not
    return self.parse_compare()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 474, in parse_compare
    expr = self.parse_math1()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 496, in parse_math1
    left = self.parse_concat()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 507, in parse_concat
    args = [self.parse_math2()]
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 517, in parse_math2
    left = self.parse_pow()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 528, in parse_pow
    left = self.parse_unary()
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 547, in parse_unary
    node = self.parse_postfix(node)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 672, in parse_postfix
    node = self.parse_subscript(node)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 705, in parse_subscript
    self.fail('expected name or number', attr_token.lineno)
  File "c:\users\raiden\documents\python projects\freespeak\venv\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected name or number
2018-08-13 22:09:23,833 INFO: Freespeak startup [in C:\Users\Raiden\Documents\Python Projects\freespeak\app\__init__.py:48]
